### PR TITLE
citation fix and other small fixes

### PR DESCRIPTION
**Pull request by MSusik**
Signed-off-by: Mateusz Susik mateuszsusik@gmail.com

I added makefile, so that citation are included with a single command.

### CODE REVIEW

File: paper/author_disambiguation.tex [hunk 1]
```diff
@@ -297,7 +297,7 @@ \subsection{Blocking}
 
 To account for these issues we propose instead to block signatures based on the
 phonetic representation of the normalized surname.
-Normalization involves stripping accents (e.g., "Jabłoński, Ł" $\rightarrow$ "Jablonski, L") and name
+Normalization involves stripping accents (e.g., "Jab\l{}o\'nski, \L{}" $\rightarrow$ "Jablonski, L") and name
```
Comment (MSusik) [inline paper/author_disambiguation.tex:308]: I can't make inputenc work, this is a workaround.

File: paper/Makefile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+all:
+	xelatex author_disambiguation
+	bibtex author_disambiguation
+	xelatex author_disambiguation.tex
```
Comment (MSusik) [inline paper/Makefile:4]: You can use `pdflatex` instead of `xelatex` as well

**Comment by glouppe** [comment]
Hmm that branch has conflicts?

**Comment by glouppe** [comment]
The Jablonski name in the paper in the current SIGIR branch appears incorrectly. Could you rebase your PR and merge it into SIGIR? It should fix it :)

**Comment by MSusik** [comment]
@glouppe  OK, I rebased.

Additionally I added hyperref.

**Pull request merged**